tensorflow/lite/core/interpreter.cc

```diff
@@ -49,8 +49,6 @@ limitations under the License.
 #endif
 #endif  // defined(__APPLE__)
 
-// TODO(b/132087118): move static_assert to c_api_internal when compiled with
-// C++.
 static_assert(sizeof(TfLiteFloat16) == sizeof(uint16_t),
```

Comment:
Please keep this one. The need to move to this is still there, the bug was closed due to bankruptcy.